src/main/java/racing/domain/Cars.java

```diff
@@ -9,13 +9,11 @@ public class Cars {
     private static final String INVALID_CAR_COUNT_MESSAGE = "자동차 경주를 진행하기 위해서는 두 대 이상의 자동차가 필요합니다.";
     private static final int MIN_CAR_COUNT = 2;
 
-    private final int carCount;
     private final List<Car> cars;
 
-    public Cars(int carCount) {
-        validateCount(carCount);
-        this.carCount = carCount;
-        this.cars = createCars();
+    public Cars(List<CarName> carNames) {
+        validateCount(carNames.size());
+        this.cars = createCars(carNames);
     }
```

Comment:
```suggestion
    public Cars(List<CarName> carNames) {
        this(createCars(carNames));
    }
    
    public Cars(List<Car> cars) {
        validateCount(cars.size());
        this.cars = cars;
    }
```
위와 같이 필드 타입과 같은 주 생성자를 통해 초기화하고, 다른 타입의 부 생성자가 주 생성자를 호출해 초기화하도록 구현하는 습관을 들여볼 것을 추천